src/Orckestra.Composer.Website/App_Insights/TelemetryInitializer.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Configuration;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.ApplicationInsights.Extensibility.Implementation;
+
+namespace Orckestra.Composer.Website.App_Insights
+{
+    public class TelemetryInitializer : ITelemetryInitializer
+    {
+        public const string C1Function = nameof(C1Function);
+
+        public void Initialize(ITelemetry telemetry)
+        {
+            TelemetryConfiguration.Active.InstrumentationKey = ConfigurationManager.AppSettings["InstrumentationKey"];
+
+            if (!(telemetry is OperationTelemetry operationTelemetry)) return;
+            var variation = (ConfigurationManager.AppSettings["Variation"] ?? "").ToUpperInvariant();
+            var operationPrefix = $"WFE{variation}";
+
+            if (string.IsNullOrEmpty(telemetry.Context.Cloud.RoleName))
+            {
+                telemetry.Context.Cloud.RoleName = operationPrefix;
+            }
+
+            var operationName = telemetry.Context.Operation.Name;
+            if (operationName.StartsWith(operationPrefix)) return;
+
+            var context = System.Web.HttpContext.Current;
+            var requestTypeKey = ComposerOvertureClient.RefAppRequestTypeKey;
```

Comment:
What this would do - it would change the name of the operation in the AppInsights to the name of the first request it does to OCS. 

But that's not necessarily a good idea - if a page reads a cart, it doesn't describe the page well.
A valid modification would be to rename Web API calls to RefApp, so the operation name is set to "WFE[CD/CM] <ApiControllerName>.<ActionName>". F.e.:
"WFECD Cart.GetShippingMethods"
"WFECD Search.AutoComplete"